src/main/fc/rc.c

```diff
@@ -823,8 +838,47 @@ bool isMotorsReversed(void)
     return reverseMotors;
 }
 
-static float quadraticBezier(float t, float p0, float p1, float p2) {
-    return (1.0f - t) * (1.0f - t) * p0 + 2.0f * (1.0f - t) * t * p1 + t * t * p2;
+// Calculates the y-coordinate of a point on a quadratic Bezier curve for a given x-coordinate.
+// It first solves for the parameter t such that Bezier_x(t) = x, using the x control points (p0x, p1x, p2x).
+// Then, it calculates the y-coordinate using the found t and the y control points (p0y, p1y, p2y).
+static float quadraticBezier(float x, float p0x, float p1x, float p2x, float p0y, float p1y, float p2y) {
```

Comment:
Part of betaflight coding styles
```suggestion
static float quadraticBezier(float x, float p0x, float p1x, float p2x, float p0y, float p1y, float p2y)
{
```